README.md

```diff
@@ -140,17 +140,17 @@ packages for Visual Studio 2012.
 
 Triton is strongly powered by [Quarkslab](https://quarkslab.com) for years but also by several strong contributors:
 
-* **Alberto Garcia Illera** - Salesforce
-* **Alexey Vishnyakov** - ISP RAS
+* **Alberto Garcia Illera** - Cruise Automation
+* [**Alexey Vishnyakov**](https://vishnya.xyz/) - ISP RAS
 * **Black Binary** - n/a
 * **Christian Heitman** - Quarkslab
-* **Daniil Kutz** - ISP RAS
+* **Daniil Kuts** - ISP RAS
 * **Florent Saudel** - Bordeaux University
 * **Jessy Campos** - n/a
-* **Jonathan Salwan** - Quarkslab
+* [**Jonathan Salwan**](http://shell-storm.org/) - Quarkslab
 * **Pierrick Brunet** - Quarkslab
 * **PixelRick** - n/a
-* **Romain Thomas** - Quarkslab
+* [**Romain Thomas**](https://www.romainthomas.fr/) - Quarkslab
 
```

Comment:
If we highlight people on this list with link we have to do it for every one (for example with @githhub link if no website/twitter exist.

Reply:
I found everyone except Florent Saudel. Do you have any links?

Reply:
Looks like he disappeared from the internet!

Reply:
Anyway, I just fixed all other issues and added new papers.